modules/cnf-performing-end-to-end-tests-junit-test-output.adoc

```diff
@@ -21,10 +21,8 @@ Use the following procedures to generate a JUnit latency test output and test fa
 [source,terminal,subs="attributes+"]
 ----
 $ podman run -v $(pwd)/:/kubeconfig:Z -v $(pwd)/junitdest:<junit_folder_path> \
--e KUBECONFIG=/kubeconfig/kubeconfig -e DISCOVERY_MODE=true -e FEATURES=performance \
-registry.redhat.io/openshift4/cnf-tests-rhel8:v{product-version} \
-/usr/bin/test-run.sh --junit <junit_folder_path> \
--ginkgo.focus="\[performance\]\ Latency\ Test"
+-e KUBECONFIG=/kubeconfig/kubeconfig registry.redhat.io/openshift4/cnf-tests-rhel8:v{product-version} \
+/usr/bin/test-run.sh --junit <junit_folder_path> 
```

Comment:
I'll add here that for the previous versions, the junit flag would create the report under the name "cnftests.xml"
so there's might be benefit to tell users to use this name in case they have some kind of automation that expects this specific report file name as the output